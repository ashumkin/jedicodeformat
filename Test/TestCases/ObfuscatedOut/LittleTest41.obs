unit littletest41;interface implementation procedure foo;begin asm
cmp al,"'"
end;end;end. 