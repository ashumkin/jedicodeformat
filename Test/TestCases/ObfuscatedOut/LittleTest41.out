unit littletest41;

interface

implementation

procedure foo;
begin
  asm
           CMP     AL,"'"
  end;
end;

end.
 